ui/src/App.tsx

```diff
@@ -45,7 +45,8 @@ const App = () => {
             }
 
         })();
-    },[]);
+        // eslint-disable-next-line
```

Comment:
Try to change:
setAddressesWithTLS(new Set([...addressesWithTLS, ...recentTLSLinks]));
to:
setAddressesWithTLS(new Set(recentTLSLinks));
I think it may solve the problem